src/runAll.js

```diff
@@ -26,7 +26,7 @@ module.exports = function runAll(packageJson, config) {
   return new Promise((resolve, reject) => {
     sgf('ACM', (err, files) => {
       if (err) {
-        reject(err)
+        return reject(err)
```

Comment:
Please add tests for those `returns`?

Reply:
I'm trying to add tests for this case but can't reproduce it, unfortunately. I'll share some work and ask your advise later or tomorrow.

Reply:
I tried the following:
```js
  it('should reject if staged-git-files returns an error', () => {
    jest.resetModules()
    jest.doMock('staged-git-files', () => (filters, cb) => {
      cb(new Error('These pretzels are making me thirsty!'))
    })
    return expect(runAll(packageJson, getConfig({}))).rejects.toMatch(
      'These pretzels are making me thirsty!'
    )
  })
```

This should have worked. But it doesn't 😞 and I can't figure out why.
@okonet I suggest we push the fix for now and deal with the test in #259.